src/Sylius/Behat/Context/Api/Shop/AddressContext.php

```diff
@@ -389,7 +389,7 @@ public function iShouldBeNotifiedThatTheProvinceNeedsToBeSpecified(): void
     {
         $response = $this->responseChecker->getResponseContent($this->addressClient->getLastResponse());
 
-        Assert::inArray(['propertyPath' => '', 'message' => 'Please select proper province.'], $response['violations']);
+        Assert::same('Please select proper province.', $response['violations'][0]['message']);
```

Comment:
maybe we could change `$response['violations'][0]['message'])` to something from `responseChecker` f.e. `hasViolationWithMessage` ?? Btw Assert::same has expected argument 2nd not 1st :D